src/components/container/container.js

```diff
@@ -251,9 +251,13 @@ export default class Container extends UIObject {
     this.trigger(Events.CONTAINER_CONTEXTMENU, this, this.name)
   }
 
-  setCurrentTime(time) {
-    this.trigger(Events.CONTAINER_SEEK, time, this.name)
-    this.playback.seek(time)
+  setCurrentTime(time, inseconds) {
```

Comment:
This will break the API and people using the latest need to update their code. 
But it only will break for people using the API if they are using just the player with the regular controls it will work just fine.